ci-operator/step-registry/telco5g/cnf-tests/telco5g-cnf-tests-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ telco5g cnf-tests commands ************"
+
+cnf_dir=$(mktemp -d -t cnf-XXXXX)
+cd "$cnf_dir" || exit 1
+
+repo="https://github.com/openshift-kni/cnf-features-deploy.git"
+branch="${PULL_BASE_REF}"
+dir="cnf-features-deploy"
+
+echo "cloning branch ${PULL_BASE_REF}"
+git clone -b $branch $repo $dir
+
+cd $dir
+KUBECONFIG=$SHARED_DIR/kubeconfig FEATURES_ENVIRONMENT="typical-baremetal" FEATURES="performance xt_u32 vrf sctp ovn" make functests-on-ci
```

Comment:
`${KUBECONFIG}` should [be defined by default][1], as long as the install step placed the generated kubeconfig in `${SHARED_DIR}/kubeconfig`.  So you should be able to drop that portion from this line.  And also from the `export` in `telco5g-gather-pao-commands.sh`.

[1]: https://docs.ci.openshift.org/docs/architecture/step-registry/#a-note-on-kubeconfig